ci-operator/config/rhpit/interop-tests/rhpit-interop-tests-main__rosa-sts-test-operator-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: rosa-sts-test-operator
+  cron: 0 6 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "operator-install", "failure_type": "all", "classification": "Provisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "operator-uninstall", "failure_type": "all", "classification": "Deprovisioning - Operators", "jira_project": "LPTOCPCI"},
+          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster (ROSA)", "jira_project": "LPTOCPCI"},
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster (AWS)", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI"},
+          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
+          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI"},
+        ]
+      FIREWATCH_DEFAULT_JIRA_PROJECT: LPTOCPCI
+      FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      OPENSHIFT_VERSION: "4.14"
+      OPERATOR1_CONFIG: ""
+    workflow: operator-aws-rosa
```

Comment:
We should find some consistency in how we are installing operators.

Our current testing for rosa-sts-hypershift is using the [install-operators ref](https://steps.ci.openshift.org/reference/install-operators) ([for example](https://github.com/openshift/release/blob/master/ci-operator/config/maistra/maistra-test-tool/maistra-maistra-test-tool-main__servicemesh-hypershift-4.13-lp-rosa-hypershift.yaml#L55)).

Here we are using the operator-aws-rosa workflow which is introducing a different way to install an operator using [operator-install](https://steps.ci.openshift.org/reference/operator-install)

Reply:
Super helpful, thanks